Tests/modules/system_related/test_sys.py

```diff
@@ -229,5 +229,9 @@ def test_getrefcount(self):
         self.assertNotEqual(0, count)
         self.assertTrue(w)
         self.assertTrue('dummy result' in str(w[0].message))
+    
+    def test_builtin_module_names(self):
+        ''' Validate properly sorted module names for issue #875 '''
```

Comment:
Maybe put the full URL for the issue instead of just the number? The repo has moved around a few times before so it saves time when trying to track down a particular issue.

Reply:
no problem! added those in